unittests/test-contracts/proxy/CMakeLists.txt

```diff
@@ -2,6 +2,5 @@ if( EOSIO_COMPILE_TEST_CONTRACTS )
    add_executable( proxy proxy proxy.cpp )
```

Comment:
change this to `add_contract( proxy proxy proxy.cpp )`

Reply:
Somehow my changes were lost with the force push of commit `fe86e15`. I am going to force push my original commits and resolve the merge conflict with a pull, because I don't know what else might have changed.

Reply:
Actually, this has gotten so messy, I think I am going to just abandon this PR and start over.

Reply:
Ugh. Sorry about that.